src/store/index.ts

```diff
@@ -168,6 +173,10 @@ export const store = createStore({
             });
             context.commit("decrementBusyCounter");
             return implementsPallet;
+        },
+
+        async RESET_BUSY(context: ActionContext<StoreI, StoreI>): Promise<void> {
+            context.commit("resetBusyCounter");
```

Comment:
I actually think we should better use the "decrementBusyCounter" and call it DECREMENT_BUSY

Reply:
got changed in the last series of commits